readme.md

```diff
@@ -177,20 +170,21 @@ require_relative 'models/artist'
 require_relative 'models/song'
 ```
 
-# ATTENTION
-From this point, I will be coding the I do portion of the new material in a wdi_app the same domain used in the active record class. You will continue to code on tunr. If at any point your code contains the words `wdi`, `instructor`, or `student` in any variant, you are doing something incorrectly.
+## Break (10 minutes / 1:05)
+
+### Important Note
 
-> Reference the code in the I do's to complete the objectives for TUNR.
+From this point, I will be coding the I do portion of the new material in a `wdi_app` the same domain used in the active record class. You will continue to code on Tunr. If at any point your code contains the words `wdi`, `instructor`, or `student` in any variant, you are doing something incorrectly.
 
```

Comment:
Not sure how I feel about these styles of code along. Another strategy code be to do one route with them for Sinatra, then they implement another, repeat....